java-rmi/src/com/baeldung/rmi/Message.java

```diff
@@ -0,0 +1,36 @@
+package com.baeldung.rmi;
+
+import java.io.Serializable;
+
+public class Message implements Serializable {
+	
```

Comment:
Isn't it best practice to declare a serialVersionUID?

Reply:
Code guidelines mentions not to write it.

Reply:
Oh, whoops! Okay, my mistake.